Services/CO.CDP.EntityVerification.Tests/Persistence/DatabasePponRepositoryTest.cs

```diff
@@ -27,25 +28,15 @@ public async Task ItFindsSavedPpon()
     public void ItRejectsAnAlreadyKnownPponId()
     {
         using var repository = PponRepository();
-        var ppon = GivenPpon(pponId: "d73c9fef3c8549e2b3999fa9ec0a5aea");
-        var anotherPpon = GivenPpon(pponId: "d73c9fef3c8549e2b3999fa9ec0a5aea");
+        var ppon = PponTestHelper.GivenPpon(pponId: "d73c9fef3c8549e2b3999fa9ec0a5aea");
+        var anotherPpon = PponTestHelper.GivenPpon(pponId: "d73c9fef3c8549e2b3999fa9ec0a5aea");
```

Comment:
Great idea to extract it. Can we give it a more meaningful name (i.e. Factories), and use a static import so it keeps the test case readable (`PponTestHelper.GivenPpon` vs `GivenPpon`)?